src/components/CourseValidationRequestForm/helpers.js

```diff
@@ -0,0 +1,49 @@
+export const getDefaultValue = (options) => (options ? options[0].key : '');
+
+export const getInitialValues = (fieldData) => {
+  const initialValues = {};
+  fieldData?.forEach((field) => {
+    initialValues[field.name] = getDefaultValue(field?.options);
+  });
+
+  return initialValues;
+};
+
+export const adaptOptions = (optionsToAdapt) => optionsToAdapt?.map((option) => ({ key: option.name.split(' ').join(''), label: option.name }));
```

Comment:
Could be an option to use replace (or replaceAll) string method instead of split and join?